docs/observability/aws/integrations/global-intelligence-cloudtrail-devops.md

```diff
@@ -0,0 +1,395 @@
+---
+id: global-intelligence-cloudtrail-devops
```

Comment:
Hi @JV0812 , are /docs/integrations/amazon-aws/global-intelligence-cloudtrail-devops and /docs/observability/aws/integrations/global-intelligence-cloudtrail-devops meant to be identical docs? 

If so, you can reference the /docs/integrations version from the /docs/observability sidebar. So sidebars.ts line item 1744 would become `integrations/amazon-aws/global-intelligence-cloudtrail-devops`.

Reply:
Both are not completely identical. The doc in ` /docs/observability/aws/integrations/global-intelligence-cloudtrail-devops` will not contain the details about collection and installation of the app. Whereas the doc under integrations folder will contain the complete info.